src/XMLimport.cpp

```diff
@@ -800,6 +800,11 @@ void XMLimport::readHostPackage(Host* pHost)
     pHost->mFORCE_GA_OFF = (attributes().value("mFORCE_GA_OFF") == "yes");
     pHost->mFORCE_SAVE_ON_EXIT = (attributes().value("mFORCE_SAVE_ON_EXIT") == "yes");
     pHost->mEnableGMCP = (attributes().value("mEnableGMCP") == "yes");
+    if (attributes().hasAttribute(QLatin1String("mEnableMSSP"))) {
+        pHost->mEnableMSSP = (attributes().value(QStringLiteral("mEnableMSSP")) == "yes");
+    } else {
+        pHost->mEnableMSSP = true;
```

Comment:
Don't have an `else` here, so in the end `Host.cpp` is the only place where we assign the default value - instead of two places. That can bite, had that happen.

Reply:
Thanks @vadi2 , I have made both changes you suggested.  I understand the flow a little better now, too.